dotNETStandard/HealthVaultProviderManagementPortal/HealthVaultProviderManagementPortal/Models/Patient/TimelineEntryViewModel.cs

```diff
@@ -41,5 +41,10 @@ public class TimelineEntryViewModel
         /// The type of schedule
         /// </summary>
         public TimelineScheduleType ScheduleType { get; set; }
+
+        /// <summary>
+        /// If this entry represents an occurrence, the ID of that occurrence
+        /// </summary>
+        public Guid? OccurrenceId { get; set; }
```

Comment:
Because not all the timeline entries are occurrences. In fact, most aren't, but I need that ID there for de-duping